### PR TITLE
Astro 11/15 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -17,6 +17,18 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.7.0 ([Release notes](cli/release-notes.md))
 
+## November 15, 2022
+
+### Additional improvements 
+
+- In the Cloud UI, the **People** tab now shows the IDs of users belonging to your Organization.
```
Comment (kwcanuck) [inline astro/release-notes.md:24]:
```suggestion
- In the Cloud UI, the **People** page now shows the IDs of users belonging to your Organization.
```
Google developer documentation style guide:

image

Microsoft Style Guide:

image

In this instance, a user selects the **People** tab to view the **People** page.

File: astro/release-notes.md [hunk 2]
```diff
@@ -17,6 +17,18 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- In the Cloud UI, you can now export a table of all users in your Organization using **Export** in the **People** tab.
```
Comment (kwcanuck) [inline astro/release-notes.md:25]:
```suggestion
- In the Cloud UI, you can now use **Export** on the **People** page to export a table of all users in your Organization.
```
Google developer documentation style guide:

image

What you're describing is a new option on a _page_ in the UI.

**Review by kwcanuck — APPROVED** [review]

File: astro/release-notes.md (hunk 2 above)
Comment (chomipi88) [reply astro/release-notes.md:25]: Export is not live in Cloud UI yet.

**Comment by chomipi88** [comment]
We added support for Updated By and Created By to the Deployments page with data as of last week. If we don't have the data we simply don't present anything.

File: astro/release-notes.md [hunk 3]
```diff
@@ -17,6 +17,17 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.7.0 ([Release notes](cli/release-notes.md))
 
+## November 15, 2022
+
+### Additional improvements 
+
+- In the Cloud UI, the **People** page now shows the IDs of users belonging to your Organization.
```
Comment (jwitz) [inline astro/release-notes.md:24]:
```suggestion
- In the Cloud UI, the **People** page now shows the IDs of users belonging to your Organization.
- In the Cloud UI, the **Deployments** page now shows the user or API key that most recently updated each Deployment and when they updated it.
```

**Review by paolaperaza — APPROVED** [review]

File: astro/release-notes.md [hunk 4]
```diff
@@ -17,6 +17,18 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- In the Cloud UI, the **Deployments** page now shows the user or API key that most recently updated each Deployment and when they updated it.
+
+### Bug fixes
+
+- Availability zone (AZ) rebalancing has been disabled for worker node pools on AWS clusters. This change should result in fewer [zombie tasks](https://airflow.apache.org/docs/apache-airflow/stable/concepts/tasks.html#zombie-undead-tasks) and less volatility across workers. AZ rebalancing is enabled for other system components on Astro.
+- The **Updated at** field for a transferred Deployment now displays the correct time.
+
```
Comment (paolaperaza) [inline astro/release-notes.md:31]:
```suggestion
- Availability zone (AZ) rebalancing has been disabled for worker node pools on AWS clusters. This change should result in fewer [zombie tasks](https://airflow.apache.org/docs/apache-airflow/stable/concepts/tasks.html#zombie-undead-tasks) and less volatility across workers. AZ rebalancing is enabled for other system components on Astro.
- The **Updated at** field for a transferred Deployment now displays the correct time.
- `astro deploy --dags` is now compatible with the triggerer and works and correctly handles deferrable tasks.
```

**Pull request merged**